File: FoodDiary.Web.Client/src/app/features/profile/models/active-session.model.ts
export type ActiveSession = {
    id: string;
    isCurrent: boolean;
    authProvider: string | null;
    browser: string | null;
    operatingSystem: string | null;
    deviceType: string | null;
    createdAtUtc: string;
    lastActiveAtUtc: string;
};

File: Modules/Identity/Application/Authentication/Models/ActiveSessionModel.cs
namespace FoodDiary.Modules.Identity.Application.Authentication.Models;

public sealed record ActiveSessionModel(
    Guid Id,
    bool IsCurrent,
    string? AuthProvider,
    string? Browser,
    string? OperatingSystem,
    string? DeviceType,
    DateTime CreatedAtUtc,
    DateTime LastActiveAtUtc);
